flink-image/ClusterFlinkImage.yaml

```diff
@@ -13,6 +13,6 @@ spec:
   apiversion: v1
   deployerVersion: v2
   docker:
-    image: claudiofahey/flink:1.10.0-2.12-1.2-W2-4-0577915d2-hadoop2.8.3
+    image: IMAGE_REF
```

Comment:
Change `IMAGE_REF` to `${IMAGE_REF}` so that it is obvious that it will be replaced.

Reply:
Changed